MAVProxy/modules/lib/wx_addlink.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+'''
+GUI for adding new links to MAVProxy
+'''
+
+import wx
+from pymavlink import mavutil
+
+from MAVProxy.modules.lib import mp_util
+from MAVProxy.modules.lib import multiproc
+
+preferred_ports = [
+    '*FTDI*',
+    "*Arduino_Mega_2560*",
+    "*3D*",
+    "*USB_to_UART*",
+    '*Ardu*',
+    '*PX4*',
+    '*Hex_*',
+    '*Holybro_*',
+    '*mRo*',
+    '*FMU*']
```

Comment:
This is going to get out-of-date with the other list.  Perhaps we drop this into a state structure or something?

Reply:
Removed and now importing preferred_ports from ``MAVProxy/modules/mavproxy_link.py``